oci/config/generate/spec.go

```diff
@@ -70,6 +66,8 @@ func (g *Generator) init() {
 // may change in the future without warning and may be invalid images).
 func New() *Generator {
 	// FIXME: Come up with some sane default.
+	// XXX VR: I don't understand the comments above as they seem to
+	// contradict. Shall we set some defaults?
```

Comment:
The warning is meant to specify that I haven't decided what sane defaults are. The zero value is /fine/ but it's possible that we would want to set some annotations in the future.